@stellar/lyra-api/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
lyra-api is for public consumption, so we should compile w babel to make sure it works for everyone